lib/moog-require.js

```diff
@@ -7,6 +7,30 @@ const importFresh = require('import-fresh');
 const resolveFrom = require('resolve-from');
 const regExpQuote = require('regexp-quote');
 
+const getDependencies = ({
+  dependencies = {},
+  devDependencies = {}
+} = {}) => {
+  return [
+    ...Object.keys(dependencies || {}),
+    ...Object.keys(devDependencies || {})
+  ];
+};
+
+const getWorkspacesDependencies = ({ workspaces = [] } = {}) => {
+  if (workspaces.length === 0) {
+    return [];
+  }
+
+  const workspacePackages = glob.sync(`{${workspaces.join(',')}}/package.json`, { follow: true });
```

Comment:
Do not make assumptions about the current working directory. Note how the code that calls `getWorkspacesDependencies` figures out what folders to look at on each iteration and pass `folder` into `getWorkspacesDependencies` so you can concatenate it with the workspace name.